.github/workflows/CI.yml

```diff
@@ -20,10 +20,19 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install imod_coupler
+      - name: Install dependencies
```

Comment:
Why do you install dependencies?

Reply:
Because the dependencies are necessary for testing